dspace-server-webapp/src/test/java/org/dspace/app/rest/CollectionGroupRestControllerIT.java

```diff
@@ -0,0 +1,2094 @@
+
+        String token = getAuthToken(admin.getEmail(), password);
+        getClient(token).perform(get("/api/core/collections/" + UUID.randomUUID() + "/bitstreamReadGroup"))
+                        .andExpect(status().isNotFound());
+    }
+
+    @Test
+    public void postCollectionDefaultBitstreamReadGroupCreateDefaultBitstreamReadGroupSuccess() throws Exception {
+
+        ObjectMapper mapper = new ObjectMapper();
+        GroupRest groupRest = new GroupRest();
+        MetadataRest metadataRest = new MetadataRest();
+        metadataRest.put("dc.description", new MetadataValueRest("testingDescription"));
+        metadataRest.put("dc.subject", new MetadataValueRest("testSubject"));
+
+        groupRest.setMetadata(metadataRest);
+
+        AtomicReference<UUID> idRef = new AtomicReference<>();
+
+        String token = getAuthToken(admin.getEmail(), password);
+        getClient(token).perform(post("/api/core/collections/" + collection.getID() + "/bitstreamReadGroup")
+                                     .content(mapper.writeValueAsBytes(groupRest))
+                                     .contentType(contentType))
+                        .andExpect(status().isCreated())
+                        .andDo(result -> idRef
+                            .set(UUID.fromString(read(result.getResponse().getContentAsString(), "$.id")))
+                        );
+        Group bitstreamReadGroup = groupService.find(context, idRef.get());
+        getClient(token).perform(get("/api/core/collections/" + collection.getID() + "/bitstreamReadGroup"))
+                        .andExpect(status().isOk())
+                        .andExpect(
+                            jsonPath("$", GroupMatcher
+                                .matchGroupEntry(bitstreamReadGroup.getID(), bitstreamReadGroup.getName())));
+
+    }
+
+    @Test
+    public void postCollectionDefaultBitstreamReadGroupCreateDefaultBitstreamReadGroupDcTitleUnprocessable()
+        throws Exception {
+
+        ObjectMapper mapper = new ObjectMapper();
+        GroupRest groupRest = new GroupRest();
+        MetadataRest metadataRest = new MetadataRest();
+        metadataRest.put("dc.description", new MetadataValueRest("testingDescription"));
+        metadataRest.put("dc.subject", new MetadataValueRest("testSubject"));
+        metadataRest.put("dc.title", new MetadataValueRest("testTitle"));
+
+        groupRest.setMetadata(metadataRest);
+
+        AtomicReference<UUID> idRef = new AtomicReference<>();
+
+        String token = getAuthToken(admin.getEmail(), password);
+        getClient(token).perform(post("/api/core/collections/" + collection.getID() + "/bitstreamReadGroup")
+                                     .content(mapper.writeValueAsBytes(groupRest))
+                                     .contentType(contentType))
+                        .andExpect(status().isUnprocessableEntity());
+
```

Comment:
double check that no group has been created and maybe that the default anonymous permission are still here